app/controllers/lunches/details/contributors.js

```diff
@@ -0,0 +1,47 @@
+var loader,
+    LoaderInstance;
+
+/**
+ *  Constructor
+ */
+( function constructor( args ) {
```

Comment:
Something is wrong with your formatting. It put's spaces before and after every bracket. This should confirm with the rest the code